renovate.json

```
@@ -216,6 +216,15 @@
       "labels": ["release_note:skip", "backport:skip"],
       "enabled": true,
       "prCreation": "immediate"
+    },
+    {
+      "groupName": "TTY Output",
+      "matchPackageNames": ["xterm", "byte-size", "@types/byte-size"],
+      "reviewers": ["awp:viz"],
```

I believe this should be changed per the Renovate [docs](https://docs.renovatebot.com/configuration-options/#reviewers):

```suggestion
      "reviewers": ["team:awp-viz"],
```